### ISSUE TITLE
bug: Fail to connect without password through browser (Cloud)

### ISSUE THREAD

**Issue by Ossanes** [shellhub-io/shellhub#4035]
### Description

When you try to connect to a device without password (user without password) you get the following message:

```
failed to get auth data from key
Connection ended
```

### Steps to Reproduce

- Open ShellHub Cloud
- Try to connect to a user without password
- Wait for the error message

### Expected Behavior

Is expected that you could connect to the device normally. Through dedicated terminal the error doesn't occur. Also if you set a user password the error doesn't occur too.

### Edition

Cloud

### Version

v0.16.0

### Related Logs

```shell
failed to get auth data from key
Connection ended
```

### Additional Information

I'm not using any Private Key or MFA setting. For these tests was used Brave and Firefox.

**Comment by haller33** [comment]
created PR to fix this issue at #4067

**Comment by haller33** [comment]
this solution will be a **breaking change**, since it allows password empty on login, as per reviewer's
